docs/en/compute-edition/31/rn/book.yml

```diff
@@ -18,6 +18,8 @@ dir: release-information
 topics:
   - name: Prisma(TM) Cloud Compute Edition Release Information
     file: release-information.adoc
+  - name: 31.02 Update 3 (Build 31.03.103)
```

Comment:
@rcaballeromx This has to be is 31.03 Update 3.
These changes are already merged by me over [here](https://github.com/hlxsites/prisma-cloud-docs/pull/225/files#diff-3c204f8f5e72064630c5fa99a6d60d4ceee96befe0909f54b9831dceca70664a)

Reply:
@divyabhushan Great, I just submitted this PR because I didn't see the changes reflected in main.